From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@kartang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

msvijayn committed:
First Try

msvijayn committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into connect-vijayn

msvijayn committed:
Draft - Metric Alerts in Preview

msvijayn committed:
Draft2 - Metric Alerts in Preview

msvijayn committed:
Draft2.1 - Metric Alerts in Preview

msvijayn committed:
Draft3 - Log and Metric Alerts in Preview

msvijayn committed:
Draftv4 - Known Issues doc

msvijayn committed:
Draftv5 - Corrections

msvijayn committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into connect-vijayn

merging as the release-connect-alerts branch has been removed; on which
the connect-vijayn was earlier based on

msvijayn committed:
Draftv5 - Corrections

msvijayn committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into connect-vijayn

msvijayn committed:
Draftv6 - appended known issues

msvijayn committed:
Draftv6 - appended known issues

msvijayn committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into connect-vijayn

msvijayn committed:
Draftv6 - edit known issues

msvijayn committed:
Draftv7 - edit final

msvijayn committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into connect-vijayn

msvijayn committed:
Draftv8 - edit knownissues

msvijayn committed:
Draftv9 - edit grammer errors

msmbaldwin committed:
Adding Financial Service Blueprint

msmbaldwin committed:
Adding Financial Service Blueprint

msmbaldwin committed:
Merge branch 'master' of https://github.com/microsoftdocs/azure-docs-pr into financial-blueprint

msvijayn committed:
Draftv10 - Doc changes Post Review Comments - Internal

msvijayn committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into connect-vijayn

Commit:
new file with update to TOC

Commit:
fix broken link

Commit:
fix acrolinx

Commit:
fixed a broken link in 9 articles

Commit:
update TOC

Commit:
ApplicationInsights: setup for conole application

Commit:
fix author

Commit:
spelling

Commit:
spelling

Commit:
spelling

Commit:
minor improvements

Commit:
fix list issues

Commit:
spelling

Commit:
more improvements

Commit:
more improvements

Commit:
up

Commit:
up

msvijayn committed:
Final for Alerts Preview

msvijayn committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into connect-vijayn

kinghorn committed:
Update operations-management-suite-service-map.md with process group information

Update to mapping overview to refer to process groups. Added a new section before machine groups with process group information. No screen shots changes in this update.

Commit:
Merge branch 'master' into lmolkova/aiconsole

msvijayn committed:
Update monitoring-overview-azure-monitor.md

Correcting invalid link

msvijayn committed:
Update monitor-alerts-unified-usage.md

Typos corrected

Commit:
post-review corrections

kgremban committed:
x509 updates

tynevi committed:
deleting unused includes

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into dec13-x509

kgremban committed:
Added screenshot and cleanup steps

kgremban committed:
code block formatting

kgremban committed:
tweaking alignment

v-viinde committed:
USER STORY 1144825 SaaS App Tutorial: IBM OpenPages

msvijayn committed:
Update monitor-alerts-unified-usage.md

typo and content change

mumian committed:
update for Dec 18th release

v-geberr committed:
wip

v-geberr committed:
wip

tynevi committed:
restoring necessary includes

tamram committed:
moving storage account options file

v-geberr committed:
wip

v-geberr committed:
wip;

v-geberr committed:
wip

v-geberr committed:
alternative build step aka workaround

cwatson-cat committed:
Updated retired Backup topic links.

v-geberr committed:
wip

v-geberr committed:
wip

tamram committed:
moving storage account options file

tamram committed:
moving storage account options file

v-geberr committed:
prereq checkmark

cwatson-cat committed:
Updated 2 backup topics to get acrolynx in the 80s

v-geberr committed:
edits

v-geberr committed:
bot csharp tutorial

v-geberr committed:
wip

MGoedtel committed:
removed update process item

MGoedtel committed:
corrected formatting of article

linda33wj committed:
Add marketo connector

msvijayn committed:
Update monitor-alerts-unified-usage.md

Updated as per comments from @rjagiewich

Commit:
images not used in \articles\azure-stack\azure-stack-powershell-install.md and \articles\azure-stack\user\azure-stack-powershell-install.md

Commit:
images not used in \articles\azure-stack\azure-stack-app-service-before-you-get-started.md

Commit:
images not used in \articles\azure-resource-manager\resource-group-create-multiple.md

Commit:
images not used in \articles\azure-resource-manager\resource-group-authoring-templates.md

Commit:
images not used in \articles\cognitive-services\custom-decision-service\custom-decision-service-get-started-app.md

Commit:
images not used in \articles\cognitive-services\custom-decision-service\custom-decision-service-get-started-browser.md

cephalin committed:
review edit

msvijayn committed:
Update toc.yml

Changed as per review comments from @rjagiewich

cephalin committed:
change pricing tier wording

cephalin committed:
staged review

KumudD committed:
reviewed articles for freshness

Commit:
New container networking article.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into container-networking

Commit:
Updates.

KumudD committed:
fixed formatting

Commit:
Updates.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into container-networking

Commit:
Updates.

Commit:
clean up scripts

mumian committed:
update

cephalin committed:
clarify SDK version

v-geberr committed:
new images

mumian committed:
update links

mumian committed:
update links

Commit:
logs

cwatson-cat committed:
fixed merge conflicts

Commit:
fixes after dev review cycle

sethmanheim committed:
Freshness updates

Commit:
fix step numbering

v-geberr committed:
changes for new UX

v-geberr committed:
update

v-geberr committed:
fixing file names

msvijayn committed:
Update monitor-alerts-unified-log.md

Inline with review comments from @rmca14

rjagiewich committed:
Merge pull request #30249 from SychevIgor/resource-group-authoring-templates

images not used in \articles\azure-resource-manager\resource-group-authoring-templates.md

Lisaco88 committed:
Merge pull request #30180 from v-geberr/1215-remove-orphan-images

wip

rjagiewich committed:
Merge pull request #30250 from SychevIgor/resource-group-create-multiple

images not used in \articles\azure-resource-manager\resource-group-create-multiple.md

rjagiewich committed:
Merge pull request #30251 from SychevIgor/azure-stack-app-service-before-you-get-started

images not used in \articles\azure-stack\azure-stack-app-service-before-you-get-started.md

rmca14 committed:
Merge pull request #30128 from mumian/1214-dec18

update for Dec 18th release

GitHubber17 committed:
Merge pull request #30237 from MGoedtel/UpdateLinuxTrouble1217

removed update process item

rjagiewich committed:
Merge pull request #30252 from SychevIgor/azure-stack-powershell-install

images not used in \articles\azure-stack\azure-stack-powershell-install.md and \articles\azure-stack\user\azure-stack-powershell-install.md

rjagiewich committed:
Merge pull request #30253 from SychevIgor/custom-decision-service-get-started-app

images not used in \articles\cognitive-services\custom-decision-service\custom-decision-service-get-started-app.md

GitHubber17 committed:
Merge pull request #30255 from cephalin/core

review edit

rjagiewich committed:
Merge pull request #30254 from SychevIgor/custom-decision-service-get-started-browser

images not used in \articles\cognitive-services\custom-decision-service\custom-decision-service-get-started-browser.md

GitHubber17 committed:
Merge pull request #30258 from cephalin/feedback

feedback

rjagiewich committed:
Merge pull request #29807 from msvijayn/connect-vijayn

Azure Alerts (Preview)

GitHubber17 committed:
Merge pull request #30262 from KumudD/freshnessDNS1218

Azure DNS content freshness review

seanli1988 committed:
Innodb_file_per_table

Innodb_file_per_table

rjagiewich committed:
Merge pull request #30196 from tamram/tamram-1215

 moving storage account options file

GitHubber17 committed:
Merge pull request #30270 from sethmanheim/relay-fresh

Freshness updates

gsacavdm committed:
Mid December update of az-gov-images

PRMerger12 committed:
Merge pull request #30278 from seanli1988/patch-1

Innodb_file_per_table

rjagiewich committed:
Merge pull request #30240 from linda33wj/master

Add marketo connector

spelluru committed:
minor cleanup

PRMerger19 committed:
Merge pull request #30279 from gsacavdm/patch-1

Mid December update of az-gov-images

cwatson-cat committed:
Fixed formating issue in troubleshooting topic.

cwatson-cat committed:
Minor upd to verb.

Lisaco88 committed:
Merge pull request #30269 from v-geberr/1218-pic-changes-how-to

new images

GitHubber17 committed:
Merge pull request #29334 from craigcaseyMSFT/v-craic1207

new file with update to TOC

PRMerger19 committed:
Merge pull request #30266 from hning86/master

clean up scripts

rjagiewich committed:
Merge pull request #29746 from lmolkova/lmolkova/aiconsole

Application Insights for Console application

PRMerger15 committed:
Merge pull request #30281 from spelluru/adf1218

minor cleanup

MGoedtel committed:
updated article to reflect changes with solution

rjagiewich committed:
Merge pull request #30121 from v-viinde/ibmopenpages

USER STORY 1144825 SaaS App Tutorial: IBM OpenPages

Commit:
Azure LBS Reference and Landing Page updates

spelluru committed:
delta copy, partioned data

spelluru committed:
acrolynx

GitHubber17 committed:
Merge pull request #30187 from cwatson-cat/12-15-ClassicBackupRetire-UpdLinks

Updated retired Backup topic links.

Commit:
removed erroneous "+" from copy/paste

Commit:
Merge pull request #30285 from spelluru/adfcopy1218

delta copy, partioned data

Commit:
ApplicationInsights console: initialize using config file; minor fixes

tynevi committed:
Merge pull request #30284 from philmea/lbs-ref-updates

Azure LBS Reference and Landing Page updates

GitHubber17 committed:
Merge pull request #30283 from MGoedtel/UpdateVMSolution1218

updated article to reflect changes with solution

v-geberr committed:
requested changes

v-geberr committed:
fixing indent

v-geberr committed:
fixed en-us in urls

rwike77 committed:
Freshness

rwike77 committed:
Freshness

tfitzmac committed:
reorganized cross rg deploy

v-geberr committed:
fuzzy search

PRMerger18 committed:
Merge pull request #30296 from rwike77/patch-6

Freshness

v-geberr committed:
c# bot template

Commit:
Incorporated PM feedback.

kgremban committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into dec13-x509

Commit:
SaaS Multi-tenant DB, Deploy. Edit pass, Acrolinx. MightyPen = GeneMi.

Ja-Dunn committed:
Merge pull request #30297 from tfitzmac/1218multiple

reorganized cross rg deploy

v-geberr committed:
edits

Commit:
review comments

Commit:
add clearer media for quickstart

PRMerger19 committed:
Merge pull request #30295 from rwike77/patch-5

Freshness

anzaman committed:
Update vpn-gateway-about-pricing-include.md

Fixed a grammatical error

Commit:
Final fix.

Commit:
add border around browser screenshots

chrissie926 committed:
add edge runtime response status

PRMerger20 committed:
Merge pull request #30300 from v-geberr/1218-fuzzy-search

fuzzy search

belmaiastar committed:
Update Users and groups screenshot

Ja-Dunn committed:
Merge pull request #30147 from v-geberr/1214-app-web-bot-csharp

wip

kgremban committed:
Acrolinx

PRMerger15 committed:
Merge pull request #30301 from anzaman/patch-27

Update vpn-gateway-about-pricing-include.md

Ja-Dunn committed:
Update container-networking.md

fixed typo in metadata description

PRMerger19 committed:
Merge pull request #30303 from chrissie926/master

add edge runtime response status

chrissie926 committed:
modify one status

Ja-Dunn committed:
Merge pull request #30299 from MightyPen/saas-20171218-multitenantdb-deploy

SaaS Multi-tenant DB, Deploy.

msmbaldwin committed:
Addressed review feedback

v-geberr committed:
fix usage of key terms

Ja-Dunn committed:
Merge pull request #30057 from kgremban/dec13-x509

x509 updates

Ja-Dunn committed:
Merge pull request #30302 from suhuruli/master

add clearer media for quickstart

markgalioto committed:
removing Classic and unnecessary Backup vault from Backup FAQ

PRMerger10 committed:
Merge pull request #30306 from chrissie926/master

modify one status

Ja-Dunn committed:
Merge pull request #28856 from msmbaldwin/financial-blueprint

Adding Financial Service Blueprint

tynevi committed:
Merge pull request #30074 from tysonn/delete-unused-includes

deleting unused includes

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Commit:
Merge pull request #30309 from v-geberr/1218-automerge

fix usage of key terms

PRMerger14 committed:
Merge pull request #30311 from markgalioto/ab-12182017

removing Classic and unnecessary Backup vault from Backup FAQ

Ja-Dunn committed:
Merge pull request #30290 from lmolkova/lmolkova/aiconsole.2

ApplicationInsights console: initialize using config file; minor fixes

markgalioto committed:
updating backup import export article for use of Classic portal

Ja-Dunn committed:
Merge pull request #30304 from linggengmsft/aad_new_portal_picture_2

Update Users and groups screenshot

v-alje committed:
Merge pull request #30314 from MicrosoftDocs/FromPublicRepo

From public repo

Ja-Dunn committed:
Merge pull request #30263 from jimdial/container-networking

Container networking

PRMerger17 committed:
Merge pull request #29842 from kinghorn/patch-1

Update operations-management-suite-service-map.md with process group …

PRMerger17 committed:
Merge pull request #30315 from markgalioto/ab-1218b

updating backup import export article for use of Classic portal
